transform.js

```diff
@@ -8,15 +8,42 @@ var logger = require('./logger');
 var STALE_DATA_THRESHOLD_MINUTES = 20;
 var SENSOR_GLUCOSE_ENTRY_TYPE = 'sgv';
 var CARELINK_TREND_TO_NIGHTSCOUT_TREND = {
-  'NONE': {'trend': 0, 'direction': 'NONE'},
-  'UP_DOUBLE': {'trend': 1, 'direction': 'DoubleUp'},
-  'UP': {'trend': 2, 'direction': 'SingleUp'},
-  'DOWN': {'trend': 6, 'direction': 'SingleDown'},
-  'DOWN_DOUBLE': {'trend': 7, 'direction': 'DoubleDown'}
+  'NONE': {
+    'trend': 0,
+    'direction': 'NONE'
+  },
+  'UP_TRIPLE': {
+    'trend': 1,
+    'direction': 'TripleUp'
+  },
+  'UP_DOUBLE': {
+    'trend': 1,
+    'direction': 'DoubleUp'
+  },
+  'UP': {
+    'trend': 2,
+    'direction': 'SingleUp'
+  },
+  'DOWN': {
+    'trend': 6,
+    'direction': 'SingleDown'
+  },
+  'DOWN_DOUBLE': {
+    'trend': 7,
+    'direction': 'DoubleDown'
+  },
+  'DOWN_TRIPLE': {
+    'trend': 7,
+    'direction': 'TripleDown'
+  }
 };
 
 function parsePumpTime(pumpTimeString, offset) {
-  return Date.parse(pumpTimeString + ' ' + offset);
+  if (process.env['MMCONNECT_SERVER'] === 'EU') {
+    return Date.parse(pumpTimeString);
+  } else {
+    return Date.parse(pumpTimeString + ' ' + offset);
```

Comment:
It's not clear to me why this behavior should be different for the EU server without sample data or tests.

Reply:
I can't explain. I'm no MMCONNECT user. @szymjaw can you explain and provide sample data and/or tests? I'm working on integrating 1.2.0 into Nightscout dev.

Reply:
In my, EU server for MMCONNECT I have sMedicalDeviceTime: "2019-03-20T09:22:07+01:00", in @mddub samples "sMedicalDeviceTime" : "Oct 21, 2015 13:51:00". I think outside Europe need add offset to correct datetime.